tests/integration/test_docker.py

```diff
@@ -54,12 +54,14 @@ def test_networking_scenario():
     Listen via netcat in one container, send a secret message to the container via another one.
     """
     image = DockerImage(THE_HELPER_IMAGE)
-    r1 = DockerRunCommand(command=["nc", "-l", "0.0.0.0", "1234"])
+    r1 = DockerRunCommand(command=["nc", "-l", "-k", "0.0.0.0", "1234"])
```

Comment:
out of curiosity, does this test run okay for you locally?

Reply:
b/c for me it fails like 50% of the time, so maybe `-k` will help

Reply:
It did succeed for me twice in a row and I didn't feel the need to do more runs, but I can try and see what's wrong there.

Reply:
I just tried running the test several times in a row, both isolated and as part of the whole test suite and it didn't fail even once. Could you possibly specify at which point the test is failing for you, and what seems to be wrong?

Reply:
It fails on the fact that the "secret" message won't reach the server netcat, so it fails on the assert. I would need to debug it with tcpdump locally and I'm lazy to do that, especially when it works for all of you and in CI.